tools/storage-advisor/src/s3Issues.go

```diff
@@ -22,10 +45,28 @@ type IssueWithCount struct {
 }
 
 type Finding struct {
-	Summary string
-	Issue   string
+	Summary          string
+	Issue            string
+	IssueDetails     string
+	Recommentation   string
+	RecommentationTf string
+}
+
+func MakeErrorFinding(summary string, issue string, issueDetails string) Finding {
+	// Copy issue detail to recomemntations, so that user sees it.
```

Comment:
Typo in `recomemntations`. I also couldn't find any evidence of existence of word "recommentation" (ending with "-tation"), but I can imagine it being some domain term.

Reply:
Thanks, that was a typo.